tests/unit/util/any.cpp

```diff
@@ -132,7 +132,7 @@ int hpx_main(variables_map& vm)
             HPX_TEST(!any1.empty());
             any any2(std::move(any1));
             HPX_TEST(!any2.empty());
-            HPX_TEST(any1.empty());
+            HPX_TEST(any1.empty()); // NOLINT
```

Comment:
What warnings did this cause?

Reply:
use after move, which this code is explicitly testing for.